v22.1/backup-and-restore-overview.md

```diff
@@ -49,6 +49,10 @@ The following table outlines SQL statements you can use to create, configure, pa
 [`RESUME SCHEDULES`](resume-schedules.html) | Resume paused backup schedules.
 [`DROP SCHEDULES`](drop-schedules.html) | Drop backup schedules.
 
+## Cockroach Cloud backup types
```

Comment:
Can you adjust the title here to match the product name https://github.com/cockroachdb/docs/blob/master/StyleGuide.md#product-names, so CockroachDB Cloud?

Also, this should be added to v22.2 of this overview page as well!